src/main/java/core/basesyntax/model/Comment.java

```diff
@@ -1,15 +1,21 @@
 package core.basesyntax.model;
 
 import java.util.List;
+import javax.persistence.Entity;
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
 
+@Entity
+@Table (name = "comments")
 public class Comment {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String content;
+    @ManyToMany
```

Comment:
```suggestion
    @OneToMany
```

Unless you plan on getting specific message by smiles

Reply:
> Unless you plan on getting specific message by smiles

It's ManyToMany since one smile may be related to many comments